### PR TITLE
Add content GET filter

### PR DESCRIPTION
**Pull request by eSKape-debug**
# Proposed Changes

Ability to download and play Tonie content

## Related Issues

[Unable to download or change Tonie content](https://github.com/mrueg/addon-teddycloud/issues/5)

### CODE REVIEW

File: teddycloud/rootfs/etc/nginx/servers/ingress.conf [hunk 1]
```diff
@@ -29,6 +29,7 @@ server {
         sub_filter '.concat("/web",' '.concat("$http_x_ingress_path/web",';
         sub_filter 'apiGetTeddyCloudApiRaw("/api/' 'apiGetTeddyCloudApiRaw("$http_x_ingress_path/api/';
         sub_filter 'apiGetTeddyCloudApiRaw(`/api/' 'apiGetTeddyCloudApiRaw(`$http_x_ingress_path/api/';
```
Comment (eSKape-debug) [inline teddycloud/rootfs/etc/nginx/servers/ingress.conf:31]: is there an actual need for duplicated filters?

Comment (mrueg) [reply teddycloud/rootfs/etc/nginx/servers/ingress.conf:31]: they are not duplicate, just different strings ` `` ` vs ` "" `

File: teddycloud/rootfs/etc/nginx/servers/ingress.conf [hunk 2]
```diff
@@ -29,6 +29,7 @@ server {
...
+	sub_filter 'apiGetTeddyCloudApiRaw(`/content/' 'apiGetTeddyCloudApiRaw(`$http_x_ingress_path/content/';
```
Comment (mrueg) [inline teddycloud/rootfs/etc/nginx/servers/ingress.conf:32]: I did not find the /content/ in the minimized js, are you sure it's in there?
I believe the content is pulled out of the json for each tonie that the API presents.

Comment (eSKape-debug) [reply teddycloud/rootfs/etc/nginx/servers/ingress.conf:32]: now i understand - ok backends builds the URL here: https://github.com/toniebox-reverse-engineering/teddycloud/blob/9e05ef8fc4a7d66f1e38db6a444c4a0d49af991f/src/handler_api.c#L2638

**Pull request closed without merging**